cis_policies_generator/src/index.ts

```diff
@@ -131,29 +156,78 @@ function parseBenchmarks(folder: string): BenchmarkSchema[] {
     });
 }
 
-function generateOutputFiles(benchmarks: BenchmarkSchema[]): void {
-    const result: any = {
-        "policies": {}
-    };
-
+function generateRulesMetadataFiles(benchmarks: BenchmarkSchema[]): void {
     for (const benchmark of benchmarks) {
         console.log("Parsed total of", benchmark.rules.length, "rules in benchmark", benchmark.filename);
-        result.policies[benchmark.filename] = {};
+        const benchmark_id = getBenchmarkAttr(benchmark.metadata, "id")
         for (let rule of benchmark.rules) {
-            result.policies[benchmark.filename][rule.rule_number] = rule;
+            const ruleNumber = rule.rule_number!.replaceAll(".", "_");
+            const rule_folder = `../bundle/compliance/${benchmark_id}/rules/cis_${ruleNumber}`
+            const metadata_file = rule_folder + "/data.yaml";
+            delete rule.rule_number;
+
+            if (fs.existsSync(rule_folder)) {
```

Comment:
we want a similar behavior for the id so we won't change the rules id's

Reply:
done